cloudvolume/lib.py

```diff
@@ -633,8 +634,11 @@ def volume(self):
   def center(self):
     return (self.minpt + self.maxpt) / 2.0
 
-  def grow(self, amt):
-    assert amt >= 0
+  def grow(self, amt: Union[int, tuple, list]):
+    if isinstance(amt, tuple) or isinstance(amt, list):
+      assert len(amt) == 3
+      amt = np.asarray(amt)
+
     self.minpt -= amt
```

Comment:
It looks like `shrink` has a check for valid Bbox that `grow` doesn't. Should this be incorporated?

Reply:
Should shrink take negative factors too?

Reply:
Oh, I did not notice that there is a shrink function!
I just added another function called adjust. please take a look.

Reply:
This seems good, but shouldn't shrink be the negative of grow?

Reply:
good point! did not realize that it should be negative!

Reply:
I have corrected this in the following commit, please take a look.